deepchecks/vision/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -229,6 +229,30 @@ def compute(self, context: Context) -> CheckResult:
 
         return CheckResult(value=values_dict, display=displays, header='Train Test Prediction Drift')
 
+    def config(self, include_version: bool = True) -> CheckConfig:
+        if isinstance(self.user_prediction_properties, list):
+            for prop in self.user_prediction_properties:
+                if 'method' not in prop or not prop['method']:
+                    raise ValueError('Each prediction property is expected to contain not emtpy "method" key')
+                if callable(prop['method']):
+                    type_name = type(self).__name__
+                    raise ValueError(
+                        f'Serialization of "{type_name}" check instance is not supported '
+                        'if custom user defined properties were passed to the "prediction_properties" '
+                        f'parameter during instance initialization. Property name: {prop["name"]}'
+                    )
+        return self._prepare_config(
```

Comment:
why not just call the parent `config` if the validation passed?

Reply:
_**point 1**_ from the description
the `prediction_properties` parameter is stored under a different name within the instance

Reply:
maybe we should change that

Reply:
what exactly?

Reply:
this
> the prediction_properties parameter is stored under a different name within the instance

Reply:
actually, somebody else did that and I did not notice after merge with the main

fixed everything else related

Reply:
so we can just call the parent `config`?

Reply:
oh you've done it, so I don't think the `_prepare_config` is needed anymore, but won't insist on this